src/core.js

```diff
@@ -309,6 +309,21 @@ $.extend( $.validator, {
 				this.element( element.parentNode );
 			}
 		},
+		onchange: function( element, event ) {
+
+			// If target element is a select, trigger focusout on change
+			if ( element.tagName.toLowerCase() === "select" ) {
+				var _this = element;
+
+				if ( !_this.form && _this.hasAttribute( "contenteditable" ) ) {
+					_this.form = $( _this ).closest( "form" )[ 0 ];
+				}
+
+				var validator = $.data( _this.form, "validator" ),
+					settings = validator.settings;
+				settings.onkeyup.call( validator, _this, event );
```

Comment:
`316 -> 324` can be shortened to:
```js
this.settings.onkeyup.call( this, element, event );
```

Also, why a `select` element should be **editable** ?